src/tracing/builder/parity.rs

```diff
@@ -501,7 +502,13 @@ where
             }
         } else {
             // account may exist or not, we need to fetch the account from the db
```

Comment:
this doc is no longer relevant if we always reload it in L487, can be removed or moved to L487 and rephrased